packages/samples/headless-ssr-commerce/app/_lib/commerce-engine-config.ts

```diff
@@ -4,33 +4,57 @@ import {
   CommerceEngineDefinitionOptions,
   CommerceEngine,
   defineProductList,
-  getSampleCommerceEngineConfiguration,
   defineQuerySummary,
 } from '@coveo/headless/ssr-commerce';
 
 type CommerceEngineConfig = CommerceEngineDefinitionOptions<
   ControllerDefinitionsMap<CommerceEngine, Controller>
 >;
 
-const configuration = {
-  ...getSampleCommerceEngineConfiguration(),
-  analytics: {
-    trackingId: 'sports-ui-samples',
-    enabled: false, // TODO: enable analytics
-  },
-};
-
 export default {
   configuration: {
-    ...configuration,
+    accessToken: 'xxc481d5de-16cb-4290-bd78-45345319d94c',
+    organizationId: 'barcasportsmcy01fvu',
+    environment: 'dev',
+    analytics: {
+      trackingId: 'sports',
+    },
```

Comment:
Why are we on barca. Can't we use searchuisamples now ?

Reply:
I think we could use the getSampleCommerceConfiguration, but as long as there isn't a context controller, we'll have to override the view.url in the config to target an existing product listing page configuration